polkadot/node/core/approval-voting/src/import.rs

```diff
@@ -711,154 +731,164 @@ pub(crate) mod tests {
-				)) => {
-					assert_eq!(h, header.parent_hash);
-					let _ = c_tx.send(Ok(session));
-				}
-			);
+				assert_matches!(
+					handle.recv().await,
+					AllMessages::RuntimeApi(RuntimeApiMessage::Request(
+						h,
+						RuntimeApiRequest::CurrentBabeEpoch(c_tx),
+					)) => {
+						assert_eq!(h, hash);
+						let _ = c_tx.send(Ok(BabeEpoch {
+							epoch_index: session as _,
+							start_slot: Slot::from(0),
+							duration: 200,
+							authorities: vec![(Sr25519Keyring::Alice.public().into(), 1)],
+							randomness: [0u8; 32],
+							config: BabeEpochConfiguration {
+								c: (1, 4),
+								allowed_slots: AllowedSlots::PrimarySlots,
+							},
+						}));
+					}
+				);
 
-			assert_matches!(
-				handle.recv().await,
-				AllMessages::RuntimeApi(RuntimeApiMessage::Request(
-					h,
-					RuntimeApiRequest::CurrentBabeEpoch(c_tx),
-				)) => {
-					assert_eq!(h, hash);
-					let _ = c_tx.send(Ok(BabeEpoch {
-						epoch_index: session as _,
-						start_slot: Slot::from(0),
-						duration: 200,
-						authorities: vec![(Sr25519Keyring::Alice.public().into(), 1)],
-						randomness: [0u8; 32],
-						config: BabeEpochConfiguration {
-							c: (1, 4),
-							allowed_slots: AllowedSlots::PrimarySlots,
-						},
-					}));
-				}
-			);
+				assert_matches!(
+					handle.recv().await,
+					AllMessages::RuntimeApi(
+						RuntimeApiMessage::Request(
+							req_block_hash,
+							RuntimeApiRequest::SessionInfo(idx, si_tx),
+						)
+					) => {
+						assert_eq!(session, idx);
+						assert_eq!(req_block_hash, hash);
+						si_tx.send(Ok(Some(session_info.clone()))).unwrap();
+					}
+				);
 
-			assert_matches!(
-				handle.recv().await,
-				AllMessages::RuntimeApi(
-					RuntimeApiMessage::Request(
-						req_block_hash,
-						RuntimeApiRequest::SessionInfo(idx, si_tx),
-					)
-				) => {
-					assert_eq!(session, idx);
-					assert_eq!(req_block_hash, hash);
-					si_tx.send(Ok(Some(session_info.clone()))).unwrap();
-				}
-			);
+				assert_matches!(
+					handle.recv().await,
+					AllMessages::RuntimeApi(
+						RuntimeApiMessage::Request(
+							req_block_hash,
+							RuntimeApiRequest::SessionExecutorParams(idx, si_tx),
+						)
+					) => {
+						assert_eq!(session, idx);
+						assert_eq!(req_block_hash, hash);
+						si_tx.send(Ok(Some(ExecutorParams::default()))).unwrap();
+					}
+				);
 
-			assert_matches!(
-				handle.recv().await,
-				AllMessages::RuntimeApi(
-					RuntimeApiMessage::Request(
-						req_block_hash,
-						RuntimeApiRequest::SessionExecutorParams(idx, si_tx),
-					)
-				) => {
-					assert_eq!(session, idx);
-					assert_eq!(req_block_hash, hash);
-					si_tx.send(Ok(Some(ExecutorParams::default()))).unwrap();
-				}
-			);
-		});
+				assert_matches!(
+					handle.recv().await,
+					AllMessages::RuntimeApi(
+						RuntimeApiMessage::Request(_, RuntimeApiRequest::NodeFeatures(_, si_tx), )
+					) => {
+						si_tx.send(Ok(NodeFeatures::repeat(enable_v2, 1))).unwrap();
```

Comment:
this could be `NodeFeatures::repeat(enable_v2, ENABLE_ASSIGNMENTS_V2 + 1)` (+1 because repeat wants a length not an index)